pkg/config/config.go

```diff
@@ -185,6 +185,9 @@ func initConfig(config Config) {
 	// Defaults to safe YAML methods in base and custom checks.
 	config.BindEnvAndSetDefault("disable_unsafe_yaml", true)
 
+	// Yaml keys which values are stripped from flare
+	config.BindEnvAndSetDefault("yaml_stripped_keys", []string{"community_string", "authKey", "privKey"})
```

Comment:
Any reason to not use a flag on the flare command to provide those keys instead?

Reply:
The issue is that you have to think about passing it every time, instead of configuring it once. That's also more useful for situations where you have configuration management (a set of hosts need those keys replaced). That's a possibility though.

Reply:
I agree, especially with the configuration management part!

Reply:
Also on Windows you might use the GUI for the flare.